cypress/integration/participantView.spec.js

```diff
@@ -13,31 +15,33 @@ describe('Participant View', () => {
 
     // Testing for freshness
 
-    cy.get('body').then(($body) => {
-      if (Cypress.$('body:contains("a day ago")').length !== 0) {
-        cy.contains('a day ago').should('exist');
-        cy.contains('14 days ago').should('exist');
-      } else {
-        cy.contains('2 days ago').should('exist');
-        cy.contains('15 days ago').should('exist');
-      }
-    });
+    // cy.get('body').then(($body) => {
+    //   if (Cypress.$('body:contains("a day ago")').length !== 0) {
+    //     cy.contains('a day ago').should('exist');
+    //     cy.contains('14 days ago').should('exist');
+    //   } else {
+    //     cy.contains('2 days ago').should('exist');
+    //     cy.contains('15 days ago').should('exist');
+    //   }
+    // });
```

Comment:
Lots of commented out code here. We should either remove it or re-add it if we need it.